clearwater-queue-manager.root/usr/share/clearwater/clearwater-queue-manager/scripts/check_node_health.py

```diff
@@ -46,7 +46,11 @@ class Status:
     CRITICAL = 2
 
 def check_status():
-    output = subprocess.check_output(['monit', 'summary'])
+    try:
+        output = subprocess.check_output(['monit', 'summary'])
+    except subprocess.CalledProcessError, e:
+        _log.error ("subprocess.check_output hit CalledProcessError, output: %s", e.output)
```

Comment:
The log would be more useful if it included what the command was and the return code - e.g. _log.error("Monit summary failed with return code {} and printed output {!r}".format(e.returncode, e.output))